java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -40,6 +40,7 @@ public interface HyperLogLogBaseCommands {
      * Estimates the cardinality of the data stored in a HyperLogLog structure for a single key or
      * calculates the combined cardinality of multiple keys by merging their HyperLogLogs temporarily.
      *
+     * @apiNote When in cluster mode, all <code>keys</code> must map to the same hash slot.
```

Comment:
maybe
```suggestion
     * @apiNote When in cluster mode, all keys in <code>keys</code> must map to the same hash slot.
```

Reply:
Team desicion with voting sayd no.
We can update all cross slot notices later if we want and find better wording.

Reply:
@shohamazon I think it's better to keep it as is.  I think the extra "keys in" makes the english awkward and too wordy.  The wording now is unambiguous.

Reply:
agreed